src/templates/static/about/job_details.jsx

```diff
@@ -106,473 +109,642 @@ const JobDetails = () => (
+                                    { text: it.L('Experience with Solidity and the development of Dapps') },
+                                    { text: it.L('Excellent understanding of Bitcoin or other cryptocurrencies') },
+                                    { text: it.L('Understanding of cryptography, including asymmetric, symmetric, hash functions, and encryption/signatures') },
+                                    { text: it.L('Familiarity with concepts such as blockchain confirmations, multisig, and HD wallets') },
+                                    { text: it.L('Knowledge of coloured coins and Ethereum tokens, including ERC20/23/223') },
+                                    { text: it.L('Experience with Bitcoin or Geth RPC APIs is a plus') },
+                                    { text: it.L('Knowledge of hashing algorithms, including SHA and scrypt') },
+                                    { text: it.L('Ability to produce high-quality, self-documenting code by using test-driven development (TDD) techniques') },
+                                    { text: it.L('Passion for Linux and other open-source platforms') },
                                 ]}
                             />
                         </div>
-                    </div>
 
-                    <div className='marketing'>
-                        <div className='invisible' id='marketing_project_coordinator'>
-                            <h1>{it.L('Marketing Project Coordinator')}</h1>
+                        <div className='invisible' id='security_researcher'>
+                            <h1>{it.L('Security Researcher')}</h1>
+
+                            <p>{it.L('[_1]\'s IT team is responsible for the design, development, and operation of our high-traffic web applications. As our Security Researcher, we expect you to stay informed about the latest security bulletins and findings, and actively monitor our software development pipeline to find and raise potential security issues.', it.website_name)}</p>
 
-                            <p>{it.L('[_1] seeks a Project Coordinator to help drive the production of promotional content and rollout of marketing campaigns, designed to engage new prospects and current customers based throughout the globe.', it.website_name)}</p>
-                            <p>{it.L('Join our team of bright, talented professionals. Work collaboratively with us to enhance and promote a high-tech platform that has been one of the most recognised in the binary options trading industry for over 15 years.')}</p>
+                            <p>{it.L('As a strong proponent of open source, we encourage publication of findings, methods, and tools via GitHub and our technical blog at https://tech.binary.com/ You will also assist our developers in understanding and patching the bugs that you find.')}</p>
```

Comment:
I assume that this is supposed to be a link? `https://tech.binary.com/`, please use a placeholder too